DP3TApp/SharedUI/Base/NSViewController.swift

```diff
@@ -35,6 +35,11 @@ class NSViewController: UIViewController {
         loadingView.stopLoading(error: error, reloadHandler: reloadHandler)
     }
 
+    // Utility function to override
+    func wasSelectedInTabbar(){
```

Comment:
Isn't the viewwillappear / didappear doing that also? why the need for a separate method?

Reply:
Yes, you`re right. I was under the impression, that this viewwillappear / didappear is also called when the app returns from the background. But this is not the case. I will change it.